packages/roosterjs-editor-core/lib/corePlugins/TypeInContainerPlugin.ts

```diff
@@ -57,12 +57,15 @@ export default class TypeInContainerPlugin implements EditorPlugin {
             // Only schedule when the range is not collapsed to catch this edge case.
             let range = this.editor.getSelectionRange();
 
+            const styledAncestor =
+                range &&
+                findClosestElementAncestor(range.startContainer, undefined /* root */, '[style]');
+
             if (
                 !range ||
                 (!this.isRangeEmpty(range) &&
-                    this.editor.contains(
-                        findClosestElementAncestor(range.startContainer, null /* root */, '[style]')
-                    ))
+                    styledAncestor &&
+                    this.editor.contains(styledAncestor))
```

Comment:
I think we should allow passing in null for contains(), so no need to check it every where.

Same for editor.select()

When pass in null, just do nothing/return false.